tools/bin/commands/release

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+NEXUS_URL="https://oss.sonatype.org"
+NEXUS_STAGING_PLUGIN_VERSION="1.6.7"
```

Comment:
I think 1.6.8 is latest, should we stick with latest?

Reply:
Sure, why not.